android/app/src/main/java/org/uwaterloo/subletr/pages/listingdetails/ListingDetailsPageViewModel.kt

```diff
@@ -51,12 +53,17 @@ class ListingDetailsPageViewModel @Inject constructor(
 		.subscribeOn(Schedulers.io())
 
 	private val imagesStream: Observable<List<Bitmap>> = imageIdsStream.map {
+		isFetchingImagesStream.onNext(true)
 		runBlocking {
 			it.map { id ->
 				async {
 					runCatching {
 						listingsApi.listingsImagesGet(id)
-					}.getOrNull()
+					}.getOrElse {
+						val currentErrors = fetchingImagesErrorsStream.value ?: emptyList()
```

Comment:
Why are we growing the list of errors?
Shouldn't the most recent set of errors be sufficient?

Also, in general, we aren't displaying the specific errors to the user right? We're just telling them that we had an error or some form.
I don't think we need to keep the `Throwables` around.

Reply:
yeah there's no UI error handling per what went wrong
I thought having all the errors would be useful later on but in that case a boolean would be enough to indicate an error occurred, wouldn't it?

Reply:
Let me think about this for a sec.

Reply:
removed this `Throwable` tracking stuff